Userland/Applications/FileManager/main.cpp

```diff
@@ -806,6 +806,22 @@ ErrorOr<int> run_in_windowed_mode(DeprecatedString const& initial_location, Depr
         window);
```

Comment:
Commit message: there's a typo ("mulitple") but I'd leave out that entire paragraph since it only makes sense if you were part of the PR discussion. As a standalone commit, it can do without that explanation.

Reply:
Removed that paragraph.